src/lib/Scenes/Collection/Screens/__tests__/CollectionArtworkPreview-tests.tsx

```diff
@@ -0,0 +1,24 @@
+import { CollectionArtworkPreviewTestsQueryRawResponse } from "__generated__/CollectionArtworkPreviewTestsQuery.graphql"
+import { renderRelayTree } from "lib/tests/renderRelayTree"
+import React from "react"
+import { graphql } from "react-relay"
+import { CollectionFixture } from "../../Components/__fixtures__/CollectionFixture"
+import { CollectionArtworkPreviewContainer as CollectionArtworkPreview } from "../CollectionArtworkPreview"
+
+jest.unmock("react-relay")
+
+it("renders properly", async () => {
+  const tree = await renderRelayTree({
+    Component: (props: any) => <CollectionArtworkPreview collection={props} {...props} />,
```

Comment:
Do you we need the typing here? If so, it might be worth providing the exact type. `CollectionArtworkPreview_collection` I think???